tools/clang/test/HLSLFileCheck/validation/WaveSizeAttrAllowed4.hlsl

```diff
@@ -0,0 +1,22 @@
+// RUN: %dxc -T cs_6_6 -E csmain1 %s | FileCheck %s
+// RUN: %dxc -T cs_6_6 -E csmain2 %s | FileCheck %s -check-prefixes=CHECK-MAIN2
+
+// another regression test that we should be sure compiles correctly
+
+// CHECK: @csmain1, !"csmain1", null, null, [[PROPS:![0-9]+]]}
+// CHECK: [[PROPS]] = !{i32 4, [[NT:![0-9]+]], i32 11, [[WS:![0-9]+]]}
+// CHECK: [[NT]] = !{i32 32, i32 1, i32 1}
+// CHECK: [[WS]] = !{i32 32}
+
+// CHECK-MAIN2: @csmain2, !"csmain2", null, null, [[PROPS:![0-9]+]]}
+// CHECK-MAIN2: [[PROPS]] = !{i32 4, [[NT:![0-9]+]], i32 11, [[WS:![0-9]+]]}
+// CHECK-MAIN2: [[NT]] = !{i32 32, i32 1, i32 1}
+// CHECK-MAIN2: [[WS]] = !{i32 32}
```

Comment:
Given that the last three lines are the same in both of these, you could add multiple prefixes to the filecheck lines and include a CHECK that contains the common parts. These are short enough that it's kind of irrelevant, but I wanted to make sure you knew of the option.